### PR TITLE
Feat create certs for remote control planes

### PR DESCRIPTION
**Pull request by kirek007**
Script connects to servers listed in control-planes.json, generate certs and store them in vault. Then download certs to local machine, split them and copy to ~/.kube/certs directory.

create_certs_for_kube.sh - is main script as a parameter you must pass common-name
create_certs_for_kube_remte.sh - is a helper script which is running on remote host (by main script)
control-planes.json - configuration for script, sample data included.

### CODE REVIEW

File: app/tests/create_certs_for_kube.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        echo "Operation at remote server failed"
+        echo "CERTIFICATES FOR ${cp_name} NOT GENERATED!"
+        continue
+    fi
+
+    echo ${SSH_OUTPUT} | jq .private_key -r > ${CERTS_PATH}/${cp_name}.priv
+    echo ${SSH_OUTPUT} | jq .issuing_ca -r > ${CERTS_PATH}/${cp_name}.isca
+    echo ${SSH_OUTPUT} | jq .certificate -r > ${CERTS_PATH}/${cp_name}.cert
+
+    echo "Certifacates for '${cp_name}' saved in ${CERTS_PATH}"
```
Comment (JulienBalestra) [inline app/tests/create_certs_for_kube.sh]: Certifacates -> Certificates

**Comment by kirek007** [comment]
Hi, I revamped script and added autoconfiguration for kubectl.

**Comment by JulienBalestra** [comment]
Nice helpers thanks !

Can we generate a variation of `app/tests/control-planes.json` with real data in a separate script by using the enjoliver API ? 

Like `curl 127.0.0.1:5000/scheduler/ip-list/kubernetes-control-plane | jq . # etc ...`

I know it will not used in production environment but could replace the `hyperkube proxy` on 127.0.0.1:8001 and fix issue #3 .

**Comment by kirek007** [comment]
@JulienBalestra, so it's not that simple. To use this script to generate certs to connect to kube api we must have a valid certificate to connect to control plane server. And as far as I remember testing.id_rsa is not in the keychain, we have to import it or improve my script to pass additional params for ssh connection. What do you think?

**Pull request merged**